### ISSUE TITLE
potential bug in imdb_wiki train_csl.py

### ISSUE THREAD

**Issue by AllenPu** [SilviaLauraPintea/reg-cls#2]
In line 636 :  for (inputs, targets, _) in tqdm(train_loader):
is that one params needs to be unpack?

**Comment by SilviaLauraPintea** [comment]
The FDS-method is not used in combination with the cls loss, as the goal is not improving regression (The paper focuses on vanilla MSE/MAE training losses for regression).
